OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -43,6 +43,7 @@ public abstract class SupportPowerInfo : PausableConditionalTraitInfo
 		public readonly string LaunchSpeechNotification = null;
 		public readonly string IncomingSound = null;
 		public readonly string IncomingSpeechNotification = null;
+		public readonly string ReinforcementsArrivedSpeechNotification = null;
```

Comment:
Why did you add this to the general `SupportPowerInfo` and not to `ParatroopersPowerInfo`? (As only the `ParatroopersPower` uses it anyway.)

Reply:
In `ParatroopersPowerInfo` there are no sound variables so I thought it would be better to have it in one place

Reply:
It's just clutter for anything else but ParatroopersPower though so I agree it would be better in PPInfo.

Reply:
Ok, I'll fix it